csrc/ir/nodes.cpp

```diff
@@ -413,19 +413,22 @@ std::vector<PolymorphicValue> UnaryOp::evaluate(
     MmaOpUtils::MatmulInputs matmul_inp;
 
     if (MmaOpUtils::matchMatmulPatterns(this, &matmul_inp)) {
-      // Inputs to MmaOp are of the shape [M, K, 1] x [1, K, N]
-      const auto a = ee.evaluate(matmul_inp.mma_lhs, known_values)
-                         .as<at::Tensor>()
-                         .squeeze(-1);
-      const auto b = ee.evaluate(matmul_inp.mma_rhs, known_values)
-                         .as<at::Tensor>()
-                         .squeeze(0);
+      // Inputs to MmaOp are of the shape [M, K] x [K, N] (matmul) / [M, K] x
+      // [N, K] (linear).
+      // Note: alpha, beta parameters are not used when linear pattern is
+      // detected (TN layout).
```

Comment:
I don't follow this comment. Why is it not a problem to ignore alpha and beta?

Reply:
I think this is because linear does not have alpha and beta params, so they should be nullptr in the match object.

Reply:
I'll rephrase the comment. alpha, beta will always be null for linear primitive.